haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
```

Comment:
I think it'd also be good to see "user" in the name, and create_initial_admin_user seems awkwardly long, so maybe `create_admin_user`?